dask_cuda/proxy_object.py

```diff
@@ -55,7 +55,7 @@ def asproxy(obj, serializers=None, subclass=None) -> "ProxyObject":
                 if callable(val):
                     val = val()
                 fixed_attr[attr] = val
-            except AttributeError:
+            except Exception:
```

Comment:
Would it make more sense to have `except (AttributeError, Type)`, i.e., the only cases being tested? I find it hard to debug this kind of catch all exceptions, and if there's a need to make something more specific for different exception types this can be very inconvenient in the future.

Reply:
Good point, in principle any error could be acceptable but keeping it to `AttributeError` and `TypeError` for now makes future debugging easier.